AEPMessagingLiveActivity/Sources/LiveActivityOrigin.swift

```diff
@@ -0,0 +1,24 @@
+/*
+ Copyright 2025 Adobe. All rights reserved.
+ This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License. You may obtain a copy
+ of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software distributed under
+ the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ OF ANY KIND, either express or implied. See the License for the specific language
+ governing permissions and limitations under the License.
+ */
+
+import Foundation
+
+/// Indicates the source of the Live Activity's creation.
+///
+/// Use this enum to distinguish whether a Live Activity was started locally by the app
+/// or remotely via push-to-start delivered by the server.
+public enum LiveActivityOrigin: String, Codable {
```

Comment:
probably should also add `@available(iOS 16.1, *)` here

Reply:
+1 @timkimadobe  We can add the availablity conditional attribute ?